src/main/content/general-references.html

```diff
@@ -1,40 +1,74 @@
 ---
 layout: doc
 title: General Reference
-type: general
 css:
 - commonConfigFeature
 - general-reference
 js: general-references
-permalink: /docs/ref/general/
+permalink: /docs/
 ---
-{% assign generalDocs = site.pages | where: 'layout', 'general-reference' | where: 'type', 'general' | sort: 'title' %}
+{% assign collectionsDoc = site.pages %}
 
-<div id="background-container">
-    <div class="row">
-        <h1 id="docs-title">Kabanero Docs</h1>
-    </div>
+{% assign docCategories = '' | split: '' %}
+
+{% for doc in collectionsDoc | where: doc.doc-category %}
+    {% assign docCategory = doc.doc-category | split: '/n' %}
+    {% assign docCategories = docCategories | concat: docCategory %}
+{% endfor %}
+
+{% assign uniqueCats =  docCategories | uniq %}
+
+
+{% assign introDoc = site.pages | where: 'title', 'Welcome to Docs' %}
+<div id="background-container" class="container-fluid">
     <div class="row">
         <!-- table of contents section -->
         <div id="toc-column" class="col-sm-2 collapsed in">
-            <div id="toc-inner">
-                {% if generalDocs.size > 0 %}
-                <div id="toc-container">
-                    <ul id="general-toc">
-                        {% for doc in generalDocs %}
-                        <li>
-                            <div role="button" href="{{ doc.url }}" tabindex="0">{{ doc.title }}</div>
-                        </li>
-                        {% endfor %}
-                    </ul>
-                </div>
-                {% else %}
-                <div>No doc found</div>
-                {% endif %}
+            <input id="doc-search" type="text" placeholder="search">
+            <hr>
+            <div class="accordion" id="docs-accordion">
+                {% for cat in uniqueCats | uniq %}
```

Comment:
same issue here with the warning/error "Expected end_of_string but found pipe in"